README.md

```diff
@@ -10,32 +10,32 @@
 
 <p align="center"><a href="https://www.nordtheme.com/ports/dircolors" target="_blank"><img src="https://raw.githubusercontent.com/arcticicestudio/nord-docs/develop/assets/images/ports/dircolors/overview.png"/></a></p>
 
-## Getting started
+# Getting started
 
 Visit the [official website][nord-home] to learn all about the [features][nord-home#intro], how to [install and activate][nord-docs-home-install] the theme, supported [data and node types][nord-docs-home-types] and more from the [official documentations][nord-docs-home].
 
-### Quick Start
+## Quick Start
 
 Nord dircolors can be installed for all application that respect the `LS_COLORS` environment variable like GNU core utils [`ls`][wiki-ls], [`tree`][wiki-tree] or modern projects like [`fd`][gh-sharkdp/fd].
 
 Download the [`dir_colors`][gh-tree-dir_colors] file and place it as `~/.dir_colors` in your [home directory][wiki-home_dir].
 
 For more details see the [official installation & activation guide][nord-docs-home-install] as well as the [GNU `dircolors` documentations][gnu-docs-dircolors].
 
-#### Latest Development State
+### Latest Development State
 
 To always use the latest development state of Nord dircolors, [clone the repository][repo] and create a [symbolic link][wiki-symlink] of the `src/dir_colors` file to `~/.dir_colors` in your [home directory][wiki-home_dir] afterwards:
 
 ```sh
-ln -sr "$PWD/src/dir_colors" "~/.dir_colors"
+ln -sr "$PWD/src/dir_colors" ~/.dir_colors
 ```
 
-#### Activation
+### Activation
```

Comment:
Again nice to see you care about correct semantics, but the heading levels have been chosen this way to ensure they fit to GitHubs rendering style. Also using a `h1` tag for “Getting Started“ would make it the title of the repository and causes mismatches when Google scrappers try to create a TOC for the search result entry. Even though there is no other `h1` the SVG banner at the top represents it and I've also started to wrap it into a `h1` tag in other projects to support scrappers/parsers.

- [ ] Please revert the heading level changes.